python/cudf_polars/cudf_polars/callback.py

```diff
@@ -66,6 +78,40 @@ def default_memory_resource(device: int) -> rmm.mr.DeviceMemoryResource:
             raise
 
 
+_SUPPORTED_PREFETCHES = {
+    "column_view::get_data",
+    "mutable_column_view::get_data",
+    "gather",
+    "hash_join",
+}
+
+
+def _enable_managed_prefetching(rmm_mode):
+    if "managed" in rmm_mode:
+        for key in _SUPPORTED_PREFETCHES:
+            pylibcudf.experimental.enable_prefetching(key)
+
+
+def _env_get_int(name, default):
+    try:
+        return int(os.getenv(name, default))
+    except (ValueError, TypeError):
+        return default
+
+
+def _env_get_bool(name, default):
```

Comment:
No need for underscores. The public interface of the module is advertised by `__all__`. 

Also, please add type annotations.

And, I would prefer if the `default` value were keyword only.

With my suggestion above to just use integers as the "flag" value for the environment variable, we can then just use:

```
def env_get_bool(name: str, *, default: bool) -> bool:
    try:
        return bool(int(os.getenv(name, default)))
    except ValueError:
        return default
```

Reply:
Fixed it.